include/comm_manager.h

```diff
@@ -180,8 +180,9 @@ class CommManager : public CommLinkInterface::ListenerInterface, public ParamLis
   void send_param_value(uint16_t param_id);
   void update_status();
   void log(CommLinkInterface::LogSeverity severity, const char * fmt, ...);
+  void log_message(CommLinkInterface::LogSeverity severity, char * text);
 
-  void send_parameter_list();
+//  void send_parameter_list();
```

Comment:
It would be better to remove old code than comment it out

Reply:
done